test/asm/underscore-in-numeric-literal.asm

```diff
@@ -0,0 +1,23 @@
+SECTION "Test", ROM0
+
+_1234::
+
+; without underscores
+	dw _1234 ; label
+	db 123, 123, 123 ; decimal
+	dw 12345 ; decimal
+	dw $abcd ; hex
+	db &200 ; octal
+	db %11110000, %10 ; binary
+TAU EQU 6.283185 ; fixed point
+	db TAU >> 24, (TAU >> 16) & $ff, (TAU >> 8) & $ff, TAU & $ff
+
+; with underscores
+	dw _1234 ; label
+	db 123, 1_23, 1__23 ; decimal
+	dw 12_345 ; decimal
+	dw $ab_cd ; hex
+	db &2_0_0_ ; octal
+	db %1111_0000, %1_0 ; binary
+TAU_ EQU 6_._283_185 ; fixed point
+	db TAU_ >> 24, (TAU_ >> 16) & $ff, (TAU_ >> 8) & $ff, TAU_ & $ff
```

Comment:
`dl` might be clearer?

Reply:
True. I was avoiding it, but only `rl` "cannot be used due to a bug", not `dl`.